modules/ROOT/pages/stats-monitoring-json.adoc

```diff
@@ -19,7 +19,7 @@ include::partial$_show_page_header_block.adoc[]
 include::partial$_define_stats_item_names.adoc[]
 
 == JSON format
-Use the `_expvars` method to request these metrics:
+Use the `_expvars` endpoint to request these metrics:
```

Comment:
```suggestion
Use the `_expvar` endpoint to request these metrics:
```

Should this be /_expvar instead ? Not sure about the style for this.

Reply:
Good catch, fixed here and throughout